src/kw_manager.sh

```diff
@@ -135,15 +146,98 @@ function check_existent_group()
   return 0
 }
 
+# This function removes a given kw_mail group and
+# all of it's references in the data base. Also
+# removes the contacts without group association
+#
+# @group_name: The name of the removed group
+#
+# Returns:
+# returns 0 if successful, non-zero otherwise
+function remove_kw_mail_group()
+{
+  local group_name="$1"
+  local group_id
+
+  if [[ "$(validate_group_name "$group_name")" ]]; then
+    return 22 #EINVAL
+  fi
+
+  if ! [[ "$(check_existent_group "$group_name")" ]]; then
+    complain 'This group doesnt exist'
+    return 22 #EINVAL
+  fi
```

Comment:
is not exactly the same, in this case the function would fail if the group doesn't exist, while when we are creating it, it would fail if the group exist. But i suppose i can remove the validate_group_name cause if the group exists then it doesn't matter if the name is valid cause it's already in the database.